Modules/WorldMapTaintWorkaround.lua

```diff
@@ -0,0 +1,30 @@
+-- This must run as early as possible.
+-- This should be independednt of Questie and all libraries.
+
+local function doWorkaround()
+    -- Blizzard's bugs #114 and #165
```

Comment:
True, #xxxx is not good as it would kind of refer to Questie repo even text infront.  I don't want to add the urls so non-developers won't go to open issues there so easily.

Reply:
Hmmm... I don't think it is a problem to add the URLs. The few users that actually open any addon files are alright to go to the issues imo